docs/user/cli/login.md

```diff
@@ -0,0 +1,137 @@
+# flightctl login
+
+Authenticate to a Flight Control service and save credentials for subsequent CLI commands.
+
+## Synopsis
+
+```shell
+flightctl login <server_url> [flags]
+```
+
+## Description
+
+The `flightctl login` command authenticates you to a Flight Control service and stores the credentials in the client configuration file (`~/.config/flightctl/client.yaml` by default).
+
+After successful authentication, all subsequent `flightctl` commands will use these stored credentials until you log out or log in to a different server.
+
+## Arguments
+
+- `<server_url>` - The URL of the Flight Control API server (e.g., `https://flightctl.example.com`)
+
+## Authentication Methods
+
+The login command supports multiple authentication methods. The method used depends on the flags provided and the server configuration:
+
+### Web-based Authentication (Browser Flow)
+
+```shell
+flightctl login <server_url> --web
+```
+
+Opens a browser window for authentication using the configured identity provider (OAuth2, OIDC, OpenShift, AAP, etc.). This is the recommended method for most users.
+
+### Token-based Authentication
+
+```shell
+flightctl login <server_url> --token=<token>
+```
+
+Authenticate using a bearer token. Useful for automation and CI/CD pipelines.
+
+### Password Authentication
+
+```shell
+flightctl login <server_url> -u <username> -p <password>
+```
+
+Authenticate using username and password credentials (PAM authentication).
+
+## Flags
+
+### Token Authentication
+
+- `-t, --token <token>` - Bearer token for direct token-based authentication
+
+### Provider-Based Authentication
+
+- `--provider <name>` - Name of the authentication provider to use (when multiple providers are configured)
+- `--show-providers` - List available authentication providers
+- `--auth-certificate-authority <path>` - Path to CA certificate file for verifying the authentication server's certificate (when different from API server)
+
+### OAuth/OIDC Flow Flags
+
+- `--web` - Use OAuth/OIDC authorization code flow (opens browser for authentication). The OAuth/OIDC provider must be configured to allow the redirect URL: `http://localhost:<callback-port>/callback`
```

Comment:
Isn't this mixing user/developer documentation?
For a "user" documentation, I'd expect the callbacks etc are properly configured.